portfolio/src/main/webapp/request.js

```diff
@@ -64,9 +65,46 @@ function onClickStartScheduling() {
  * Updates the UI to show the results of a query.
  */
 function handleScheduledTaskArray(scheduledTaskArray) {
-  const resultElement = document.getElementById('scheduled-task-list');
+  const resultElement = document.getElementById('schedule-result-list');
   resultElement.innerHTML = '';
-  scheduledTaskArray.forEach(addScheduledTaskToDom, resultElement);
+  scheduledTaskArray.forEach(addScheduledTaskToDom);
+}
+
+/**
+ * Returns a JSON of the latest scheduled tasks pulled from the DOM.
+ */
+function collectAllScheduledTasks() {
+  const allScheduledTasks = new Array();
+  // Get scheduled task list element to run through cards collecting data.
+  const scheduledTaskList = document.getElementById('schedule-result-list');
+
+  scheduledTaskList.childNodes.forEach((scheduledTaskCard) => {
+    const cardBody = scheduledTaskCard.childNodes[0];
+
+    const scheduledTaskName =
+        cardBody.childNodes[0].getAttribute('data-task-name');
+    const scheduledTaskScheduledTime =
+        cardBody.childNodes[1].getAttribute('data-task-time');
+    const scheduledTaskDurationMinutes =
+        parseInt(cardBody.childNodes[2].getAttribute('data-task-duration'));
+    const scheduledTaskDescription =
+        cardBody.childNodes[3].getAttribute('data-task-description');
+    const scheduledTaskPriority =
+        parseInt(cardBody.childNodes[4].getAttribute('data-task-priority'));
+
+    var scheduledTask = {};
+    const task = new Task(
+        scheduledTaskName,
+        scheduledTaskDescription,
+        scheduledTaskDurationMinutes,
+        scheduledTaskPriority
+        );
```

Comment:
nit: This end paren can go to the end of the previous line.

ex:
scheduledTaskPriority);

Reply:
Done.